src/test/java/com/werfen/report/GridReportTest.java

```diff
@@ -26,6 +26,7 @@
     private static final String COLUMN_PREFIX_TRANSLATION = "column ";
     private static final String COORDINATES_SEPARATOR = ".";
     private static final String GOLDEN_SUFFIX = "_golden";
+    private static final String GOLDEN_PATH = "src/test/resources/golden/";
```

Comment:
I think it would be good to harmonize the terminology for the test resources, so instead of using the word "golden", it would be more meaningful to use "expected". Moreover, the files can be named using the following pattern:
* src/test/resources/pdf/testA.expected.pdf
* src/test/resources/pdf/testB.expected.pdf
* src/test/resources/xlsx/testA.expected.pdf
* src/test/resources/xlsx/testB.expected.pdf
* ...

Then, the files generated by each test can be placed in the same location, but using the suffix "actual". For instance:
* src/test/resources/pdf/testA.actual.pdf
* src/test/resources/pdf/testB.actual.pdf
* src/test/resources/xlsx/testA.actual.pdf
* src/test/resources/xlsx/testB.actual.pdf
* ...

Reply:
Need to check with @alfonsserra,